evil-commands.el

```diff
@@ -2924,6 +2924,26 @@ for the last window in each frame."
                   (error nil)))
             wins))))
 
+(evil-define-command evil-wipeout-buffer (buffer &optional bang)
+  "Deletes a buffer. Also clears related jump marks.
+All windows currently showing this buffer will be closed except
+for the last window in each frame."
+  (interactive "<b><!>")
+  (let* ((ring (make-ring evil-jumps-max-length))
```

Comment:
Don't use `let*` where `let` is sufficient.

Reply:
there is a variable dependency